cmd/build.go

```diff
@@ -36,19 +40,35 @@ func setupBuildCommand() *cobra.Command {
 func buildCommandAction(cmd *cobra.Command, args []string) error {
 	cmd.Println("Build the package")
 
-	target, err := docs.UpdateReadme()
-	if err != nil {
-		return errors.Wrapf(err, "updating %s file failed", docs.ReadmeFile)
+	packageRootPath, found, err := packages.FindPackageRoot()
+	if !found {
+		return errors.New("package root not found")
 	}
-	if target != "" {
-		cmd.Printf("%s file rendered: %s\n", docs.ReadmeFile, target)
+	if err != nil {
+		return errors.Wrap(err, "locating package root failed")
 	}
 
-	target, err = builder.BuildPackage()
+	readmeFiles, err := ioutil.ReadDir(filepath.Join(packageRootPath, "_dev", "build", "docs"))
```

Comment:
Wouldn't this change break backwards compatibility? Older packages that define their package READMEs under `<package root>/_dev/build/README.md` would not get processed correctly by `elastic-package build` any more, right?

Reply:
Hmmm I thought older packages also have README.md under `<package root>/_dev/build/docs/README.md`?
This change will still build `README.md` file and on top of that, build the rest of the `.md` files.

Reply:
I think the current code preserves the backwards-compatibility rule. It just renders another .md templates.

Reply:
Yes, you're right. I was thinking existing packages' READMEs go under `_dev/build` but I forgot the `docs` sub-folder at the end. Carry on, nothing to see here 😊.